{"name": "test_login_params[data_for_login1]", "status": "passed", "attachments": [{"name": "log", "source": "8b6d2110-fc19-4dd3-834f-b14f5373c167-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data_for_login", "value": "('adelukesh1@gmail.com', '12345', 'Fail')"}], "start": 1745744032198, "stop": 1745744034720, "uuid": "e3304de8-e273-478b-b9f9-a7a5d3623f14", "historyId": "4de2338e3b208b25a3a2b07a9b7ad841", "testCaseId": "5fe6fe3249a01c883d8efece63f2c386", "fullName": "test_login_params.TestLoginParams#test_login_params", "labels": [{"name": "severity", "value": "critical"}, {"name": "tag", "value": "regression"}, {"name": "tag", "value": "sanity"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup', 'log_on_failure')"}, {"name": "suite", "value": "test_login_params"}, {"name": "subSuite", "value": "TestLoginParams"}, {"name": "host", "value": "lukesh"}, {"name": "thread", "value": "11420-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_login_params"}]}